VimeoNetworking/Sources/Request+Cache.swift

```diff
@@ -32,12 +32,10 @@ public extension Request
     /// Generates a unique cache key for a request, taking into account endpoint and parameters
     var cacheKey: String
     {
-        var cacheKey = "cached" + self.path
+        let url = NSURL(string: self.path)
+        let urlPath = url?.path ?? ""
         
-        if let parameters = self.parameters
-        {
-            cacheKey = cacheKey + "." + String(parameters.description.hashValue)
-        }
+        var cacheKey = "cached" + urlPath + "." + String(self.path.hashValue)
```

Comment:
1) What we tried to achieve in the past was to generate a unique cached key based on the parameters. However, the parameter doesn't change when you do a paginated request.

2) What we want to do instead: based on the `self.path` that is equal to the `uri` generate a uniqueKey. That should be enough because the uri contains the `real path + the queryItems (= variable when you paginate)`.